src/components/generic/Header.tsx

```diff
@@ -0,0 +1,105 @@
+import { NavigationProp, useNavigation } from '@react-navigation/native';
+import React from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import COLORS from '../../assets/colors';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+interface headerProps {
+  title: string;
+  hasHomeButton?: boolean;
+  hasBackButton?: boolean;
+  hasHelpButton?: boolean;
+}
+
+const Header = ({ title, hasHomeButton, hasBackButton, hasHelpButton }: headerProps) => {
+  const navigation = useNavigation();
+
+  return (
+    <View style={styles.headerBar}>
+      <View style={styles.headerTop}>
+        {hasHomeButton ? (
+          <TouchableOpacity
+            accessibilityLabel="Home knop"
+            accessibilityHint="Als je naar het hoofdscherm wilt navigeren, druk dan op deze knop."
+            onPress={() => handleHomeButton(navigation)}
+          >
+            <Icon name="home" size={40} color={COLORS.white} />
+          </TouchableOpacity>
+        ) : (
+          hasBackButton && (
+            <TouchableOpacity
+              accessibilityLabel="Terug knop"
+              accessibilityHint="Als je naar het vorige scherm wilt navigeren, druk dan op deze knop."
```

Comment:
"Terug naar XXX" en dan misschien kijken of je de titel van het scherm kan meegeven.